hack/run-critest.sh

```diff
@@ -14,19 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Run critest with default dockershim.
+# Run critest with pouch.
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
-# Install nsenter
-docker run --rm -v /usr/local/bin:/target jpetazzo/nsenter
+# Start pouch cri server first
+/usr/local/bin/kubelet --v=3 --logtostderr --container-runtime=remote \
+--container-runtime-endpoint=unix:///var/run/pouchcri.sock \
+--image-service-endpoint=unix:///var/run/pouchcri.sock \
+--docker=unix:///var/run/pouchd.sock &
```

Comment:
It is unnecessary to run kubelet when running critest with pouch-cri. 

Change the container runtime endpoint of critest, like `critest --runtime-endpoint=/var/run/pouchcri.sock -ginkgo.skip="runtime should support reopening container log" -parallel 8`. It enables critest communicate with pouch-cri 😄.

Reply:
Well, I have replaced kubelet with critest for testing directly.